gs/src/imgui_impl_opengl3.cpp

```diff
@@ -270,7 +270,11 @@ void    ImGui_ImplOpenGL3_RenderDrawData(ImDrawData* draw_data, bool rotate)
                     }
 
                     // Bind texture, Draw
+#if defined(RASPBERRY_PI)
                     uint32_t texId = (uint32_t)pcmd->TextureId;
+#else
+                    uint64_t texId = (uint64_t)pcmd->TextureId;
```

Comment:
Could we use uint64_t for the pi as well? Or size_t?

Reply:
TextureID comes from imgui  struct ImDrawCmd. I think it's the library definition.

Reply:
Please check [Fix void * type cast issue on 32/64bit systems](https://github.com/jeanlemotan/esp32-cam-fpv/pull/25/commits/47efa1a868e0acc25ce7a8deafda53fff4beb14e)

Reply:
intptr_t  is a good idea.